README-zh-Hans.md

```diff
@@ -110,10 +112,37 @@ docker build -t colossalai ./docker
 docker run -ti --gpus all --rm --ipc=host colossalai bash
 ```
 
+
+## 社区
+欢迎通过[论坛](https://github.com/hpcaitech/ColossalAI/discussions)
+[Slack](https://join.slack.com/t/colossalaiworkspace/shared_invite/zt-z7b26eeb-CBp7jouvu~r0~lcFzX832w)
+或 [微信](./docs/images/WeChat.png "qrcode")
+加入Colossal-AI社区，与我们分享你的建议和问题。
+
+
 ## 做出贡献
 
 欢迎为该项目做出贡献，请参阅[贡献指南](./CONTRIBUTING.md)。
 
+真诚感谢所有贡献者！
```

Comment:
Is the order of the contributor list is randomly shuffled?

Reply:
I don't think so. The list of contributors is often crawled from [here](https://github.com/hpcaitech/ColossalAI/graphs/contributors) and follows its order.

Reply:
I think we should shuffle the list, just like taichi. If we always present our inside contributors at first, it may be viewed as a kind of show-off. If we shuffle the list and hide our credit, it will make outside contributor more motivated :)

Reply:
The avatar list is shuffled in the latest `public_asset` repo.